README.md

```diff
@@ -116,3 +116,28 @@ The OpenShift descriptor will automatically create `Route`s for Prometheus and J
    > For production-ready Jaeger instances, please see the [Jaeger Operator documentation](https://operatorhub.io/operator/jaeger) for how to properly deploy and configure production-ready instances.
    >
    > For production-ready OpenTelemetry Collector instances, please see the [OpenTelemetry Operator documentation](https://operatorhub.io/operator/opentelemetry-operator) for how to properly deploy and configure production-ready instances.
+
+### Jaeger
+
+Telemetry data is exported to a Jaeger instance, which can be analyzed and visualized at `http://localhost:16686`. 
+By now you've performed a few battles, so let's analyze the telemetry data.
+Open the Jaeger UI by going to `http://localhost:16686`.
+
+![Jaeger Filters](images/jaeger-1-inputs.png)
+
+Now, let's analyze the traces for when requesting new fighters.
+When clicking the **New Fighters** button in the Superheroes UI, the browser makes an HTTP request to the `/api/fights/randomfighters` endpoint within the `rest-fights` application.
+In the Jaeger UI, select `rest-fights` for the Service and `/api/fights/randomfighters` for the Operation, then click **Find Traces**.
+You should see all the traces corresponding to the request of getting new fighters.
+
+![Jaeger Filters](images/jaeger-2-list-traces.png)
+
+Then, select one trace.
+A trace consists of a series of spans.
+Each span is a time interval representing a unit of work.
+Spans can have a parent/child relationship and form a hierarchy.
+You can see that each trace contains 12 total spans:
```

Comment:
Its actually 14 total spans (I know where you got that number from - I added JDBC tracing last week and the place where you got that was written before I made that change :) )